fontir/src/piecewise_linear_map.rs

```diff
@@ -0,0 +1,112 @@
+//! Remaps values using a series of linear mappings.
+//!
+//! Useful for things like designspace : userspace mapping.
```

Comment:
it's the other way around, `userspace : designspace mapping`

Reply:
I updated the comment, to my understanding you would map design : user and not, at least not when going from source to IR, likely the other way around?

Reply:
(lightbulb) now I see, you meant https://fonttools.readthedocs.io/en/latest/designspaceLib/xml.html#map-element maps the other way around? 

I would think for building IR we want userspace coords not design coords so the comment stands. That probably means I'm populating it backwards in #47.